progress/react/icons/src/types.ts

```diff
@@ -14,6 +14,7 @@ export type ProgressIconProps = SVGAttributes<SVGElement> & {
         progressIconLabel?: string;
         centered?: string;
     };
+    labelProps?: DetailedHTMLProps<HTMLAttributes<HTMLDivElement>, HTMLDivElement>;
```

Comment:
I'm still not seeing the use case for `labelProps`. It makes it harder to override the typical style changes:
```
<ProgressIcon labelProps={{style: {color: 'red', fontSize: 32}}} />
// vs
<ProgressIcon labelColor={'red'} labelSize={32} />
```

If we're going to have it, it should be in the Readme API and it should be (it's a span not a div):
```
labelProps: HTMLAttributes<HTMLSpanElement>
```
and the spread should come after the props that we set (or they get overridden by our props and do nothing).

For the purposes of doing anything more complicated than size/color, I think a user would be better off just building their own label and using it alongside the (unlabeled) progress icon 🤷

Reply:
Ah, for some reason I thought it was a Typography component

Reply:
I agree with you @joebochill this prop seems like overkill; I'm having a hard time seeing people use it.  I've removed it entirely.